### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by Jaymelfah**
###  In this project, I used relational database to create the initial data structure for a Vet clinic.

> I created 3 tables to store information about the various animals, their owners, and their species. 
> I wrote queries using JOIN and aggregates to analyze the data from multiple tables.
> I added foreign key columns to the existing animals' table to model one-to-many relationships.

Screenshot of query operations:

Screenshot 2022-12-06 at 1 10 51 PM
Screenshot 2022-12-06 at 1 11 35 PM

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -33,10 +33,27 @@ COMMIT;
...
+
+SELECT species.name, COUNT(*) FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.name;
+
+SELECT animals.name AS "Animal", owners.full_name AS "Owner", species.name AS "Species" FROM animals INNER JOIN owners ON animals.owner_id = owners.id INNER JOIN species 
+ON animals.species_id = species.id WHERE full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+
+SELECT animals.name AS "Animal", owners.full_name AS "Owner" FROM animals INNER JOIN owners ON animals.owner_id = owners.id WHERE escape_attempts = 0 AND 
+full_name = 'Dean Winchester';
+
+SELECT full_name, COUNT(*) FROM owners INNER JOIN animals ON owners.id = animals.owner_id GROUP BY full_name ORDER BY COUNT(*) DESC LIMIT 1;
```
Comment (Deepakdanger) [inline queries.sql]: - [ ] Here we have to join all the query via "LEFT JOIN"

File: schema.sql [hunk 2]
```diff
@@ -9,3 +9,17 @@ CREATE TABLE animals (
  weight_kg NUMERIC,
  species TEXT
 );
+
+
+CREATE TABLE owners(
+id SERIAL NOT NULL PRIMARY KEY,
+full_name VARCHAR,
+age INT);
```
Comment (Deepakdanger) [inline schema.sql:17]: - [ ] KIndly add `PRIMARY KEY (id)` in tale creation.
```sql
   CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
   );
```

File: schema.sql [hunk 3]
```diff
@@ -9,3 +9,17 @@ CREATE TABLE animals (
...
+
+CREATE TABLE species(
+id SERIAL NOT NULL PRIMARY KEY,
+name VARCHAR);
```
Comment (Deepakdanger) [inline schema.sql:21]: - [ ] KIndly add `PRIMARY KEY (id)` in tale creation.
```sql
CREATE TABLE species (
   id int NOT NULL IDENTITY(1,1),
   name varchar(255),
   PRIMARY KEY (id)
   );
```

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Jaymelfah , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (Jaymelfah) [reply schema.sql:17]: I already added the PRIMARY KEY on the same line as the id

File: schema.sql (hunk 3 above)
Comment (Jaymelfah) [reply schema.sql:21]: I already added the PRIMARY KEY on the same line as the id

File: queries.sql (hunk 1 above)
Comment (thecodechaser) [inline queries.sql]: - [ ] As requested by the previous reviewer, Please make sure to join all the queries via "LEFT JOIN".

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Jaymelfah👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Jaymelfah) [reply queries.sql]: Thanks for the feedback.  Screenshot also attached

Screenshot 2022-12-06 at 2 18 07 PM

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Jaymelfah👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**